internal/report/report.go

```diff
@@ -237,9 +296,12 @@ func newPolicyReportResult( //nolint:funlen
 		Timestamp:       timestamp, // time the result was computed
 		Result:          result,    // pass, fail, error
 		Scored:          scored,
-		Subjects:        []*v1.ObjectReference{resourceObjectReference}, // reference to object evaluated
+		Subjects:        []*v1.ObjectReference{&resourceObjectReference}, // reference to object evaluated
 		SubjectSelector: &metav1.LabelSelector{},
 		Description:     description, // output message of the policy
-		Properties:      map[string]string{},
+		// The policy resource version is used to check if the same result can used
+		// in the next scan
+		// https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency
+		Properties: map[string]string{PropertyPolicyResourceVersion: policy.GetResourceVersion()},
```

Comment:
I'm not sure this is correct; I don't think we should save policy resource versions on the result.
Properties could be end-user readable.

Reply:
I think we should use the `ResourceRevision` (see comment below), but we should not store that inside of the `Properties`. Rather, we should use the dedicated field inside of the `ObjectReference` struct: https://pkg.go.dev/k8s.io/api/core/v1#ObjectReference